apps/sap-commerce-cloud/frontend/src/components/Field/Field.spec.tsx

```diff
@@ -16,16 +16,4 @@ describe('Field', () => {
     );
     expect(container).toMatchSnapshot();
   });
-
-  it('should update state value', () => {
```

Comment:
These tests were failing with `TypeError: default is not a constructor`. I opted to just remove them since our tests should mimic behavior to trigger the function rather than call the internal methods directly